app/src/main/java/com/globant/tutoria/subscriberssearch/model/SubscribersModel.java

```diff
@@ -0,0 +1,180 @@
+package com.globant.tutoria.subscriberssearch.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by juan.herrera@globant.com on 11/08/2016.
+ */
+public class SubscribersModel {
+
+    @SerializedName("login")
```

Comment:
No need to use `SerializedName` when the name on the JSON field is the same of the attribute.